src/utils/engines.js

```diff
@@ -282,9 +289,12 @@ export async function update(name) {
 
       // Save the new engine to memory and storage
       saveToMemory(name, engine);
-      saveToStorage(name);
+      saveToStorage(name, data.engines[ENGINE_VERSION].checksum);
```

Comment:
Should this have error handling? Also, if all other usage of saveToStorage just continue, but log errors, would introducing a "trySaveToStorage" function make sense? Which handles the logging, but is fire-and-forget, providing no guarantees beside best-effort.

Reply:
It has, but inside the `saveToStorage` (try/catch). The name follows the convention of other targets (`saveToMemory`, `saveTo...`, `loadFrom....`). Your proposition might be valid, but I would keep this PR focused on the DNR fixes.